src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -185,11 +185,11 @@ const CpsOnwardJourney = ({
         {children}
       </Wrapper>
     ) : (
-      <LegacyGridWrapper data-e2e={labelId} {...a11yAttributes}>
-        <LegacyGridItemConstrainedLarge>
+      <GridWrapper data-e2e={labelId} {...a11yAttributes}>
```

Comment:
This element has a 2rem `padding-bottom` style on live that persists across all breakpoints:
![Screenshot 2020-10-26 at 16 02 33](https://user-images.githubusercontent.com/9042672/97196735-b72a2500-17a4-11eb-8ee3-9abc9aa0e620.png)

You can see it disappearing in ln6 of the snap: [src/app/containers/CpsOnwardJourney/__snapshots__/index.test.jsx.snap](https://github.com/bbc/simorgh/pull/8194/files#diff-8ffb0649a2a22986255f1fd00bad9f0be2eafcda4cf109f4e1d6aa0aee9dee9bL6)

I think you could just make use of emotion's `css` prop here to apply the style:

```suggestion
      <GridWrapper 
        data-e2e={labelId} 
        {...a11yAttributes} 
        css={css`
          padding-bottom: 2rem;
        `}
      >
```